application.js

```diff
@@ -46,31 +48,85 @@ var sessionOptions = {
   }
 };
 
+/**
+ * Gets mongodb connection information from $fh.db
+ * Use Data Browser > Upgrade Database to get the connection information available
+ */
+function mongoInit(cb) {
+  $fh.db({
+    act: 'connectionString'
+  }, function(err, connectionString) {
+    if (err) {
+      return cb(err);
+    }
+    if (!connectionString) {
```

Comment:
If there is no connection string, it should return an error advising to upgrade the database.